core/src/main/scala/algebra/Eq.scala

```diff
@@ -69,6 +69,11 @@ object Eq extends EqFunctions {
       def eqv(x: A, y: A) = f(x, y)
     }
 
+  def fromAll[A](fs : ((A,A) => Boolean)*) : Eq[A] =
```

Comment:
That said, I doubt we need `@sp`. You'd almost never want it here right? This is for classes/traits I assume.